### ISSUE TITLE
Detection of potential omissions of missing values in expressions shouldn't be flagged when missing value conditions are explicitly stated in expressions

### ISSUE THREAD

**Issue by luisesanmartin** [worldbank/stata-linter#49]
For example, lines like these:
```{stata}
keep if var != 0 & var != .
keep if var != 0 & var ~= .
keep if var != 0 & !missing(var)
```
shouldn't trigger the issue:
```{stata}
Are you taking missing values into account properly? (Remember that "a != 0" includes cases where a is missing.)
```
